### DISCUSSION TITLE
buddhism/2021/docs_2021/sadhana/liberation/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#65] [General]
# buddhism/2021/docs_2021/sadhana/liberation/

https://shanenull.com/buddhism/2021/docs_2021/sadhana/liberation/

**Comment by shane0** [comment]
Look directly at whatever arises and do not attempt to alter it. Simply rest. All the stages of creation and completion, mantra repetition and visualization are included in this. You must learn to distinguish this practice from distraction. If you are undistracted and naturally at ease, this is awareness, where there is neither gain nor loss. However, if distracted—even slightly—you may give into like and dislike and the accumulation of positive and negative action, the fuel to wander in saṃsāra. Learn to make this distinction between awareness and mind or thought; rely on the stability of the former and purify the latter, gathered as they are like clouds.

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2023/samu/

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2023/liberation/
